CondTools/L1TriggerExt/interface/L1ConfigOnlineProdBaseExt.h

```diff
@@ -156,9 +168,10 @@ bool L1ConfigOnlineProdBaseExt<TRcd, TData>::getObjectKey(const TRcd& record, st
 
   // If L1TriggerKeyExt is invalid, then all configuration objects are
   // already in ORCON.
-  edm::ESHandle<L1TriggerKeyExt> key;
+  // edm::ESHandle<L1TriggerKeyExt> key_token;
+  L1TriggerKeyExt key;
   try {
-    keyRcd.get(key);
+    key = keyRcd.get(key_token);
```

Comment:
Does the object really need to be copied?

Reply:
Again I don't know the idea behind this... maybe avoid calling Record::get() twice ? I can remove the copy if necessary

Reply:
It is up to you/L1T, but in general we try avoid unnecessary copies because the copy wastes CPU cycles and memory. The previous implementation (with `ESHandle`) did not incur a copy.